### PR TITLE
Update reporting.md

### PR DESCRIPTION
**Pull request by abrotman**
adding new template/sample for the report body.

### CODE REVIEW

File: reporting.md [hunk 1]
```diff
@@ -234,7 +234,8 @@ Aggregate reports contain the following fields:
 
 Note that the failure types are non-exclusive; an aggregate report may contain
 overlapping `counts` of failure types when a single send attempt encountered
-multiple errors.
+multiple errors.  Additionally, in the event that only one policy is being
```
Comment (danmarg) [inline reporting.md]: I can't parse this last sentence. don't you mean if *multiple* policies are being reported?

File: reporting.md [hunk 2]
```diff
@@ -327,35 +328,39 @@ The JSON schema is derived from the HPKP JSON schema [@?RFC7469] (cf. Section 3)
 
 ```
 {
-  "organization-name": organization-name,
+  "organization-name": "organization - name",
```
Comment (danmarg) [inline reporting.md]: Why this change (and those below)? I mean, why add quotes and all?

File: reporting.md (hunk 1 above)
Comment (abrotman) [reply reporting.md]: No, I meant to say that even if there is only one, please still include the array of "policy" inside of "polciies".

File: reporting.md (hunk 2 above)
Comment (abrotman) [reply reporting.md]: To put the strings in quotes?  AFAIK, that's how JSON expects them to be, and I would think someone wouldn't want to put a bare string after the colon.  It can be reverted.

File: reporting.md (hunk 1 above)
Comment (danmarg) [reply reporting.md]: I don't think it's necessary to say this at all, then. If it's an array, it should always be an array. Isn't that the point of the schema?

File: reporting.md (hunk 2 above)
Comment (danmarg) [reply reporting.md]: Agreed. But this is a schema, not an example report. I thought it made sense that "date-time" and "email-address" and so forth were variables standing in for the real thing. That's how it's written in the referenced https://tools.ietf.org/html/rfc7469#section-3.

File: reporting.md (hunk 1 above)
Comment (abrotman) [reply reporting.md]: Yes, but I didn't want a reporter to think it was okay to omit it if the length was one.

File: reporting.md (hunk 2 above)
Comment (abrotman) [reply reporting.md]: Ugh, crap, I thought you were commenting on the sample.  Yeah, we can remove those quotes.
